lib/rucio/core/transfer.py

```diff
@@ -811,41 +811,6 @@ def set_transfer_update_time(external_host, transfer_id, update_time=datetime.da
         raise UnsupportedOperation("Transfer %s doesn't exist or its status is not submitted." % transfer_id)
 
 
-def query_latest(external_host, state, last_nhours=1, logger=logging.log):
-    """
-    Query the latest transfers in last n hours with state.
-    :param external_host:  FTS host name as a string.
-    :param state:          FTS job state as a string or a dictionary.
-    :param last_nhours:    Latest n hours as an integer.
-    :param logger:         Optional decorated logger that can be passed from the calling daemons or servers.
-    :returns:              Requests status information as a dictionary.
-    """
-
-    record_counter('core.request.query_latest')
-
-    start_time = time.time()
-    resps = FTS3Transfertool(external_host=external_host).query_latest(state=state, last_nhours=last_nhours)
```

Comment:
I wonder if we can remove the `query_latest` in the transfertool as well.

Reply:
Done